### DISCUSSION TITLE
output to readb ?

### DISCUSSION THREAD

**Discussion by madjuju** [szpajder/dumphfdl#36] [Q&A]
Hello
dumphfdl ok on my raspberry with rsp1A.

have another pi with reads & tar1090 combine

how can l send dumphfdl tcp/udp to reads ?

did not undertound what dumphfdl --output help said

**Comment by dg9bja** [comment]
Guess you mean readsb?
You need following parameters in your combine1090 config:
--jaero-timeout=180 --net-sbs-jaero-in-port=29106

And add a command line like this to dumphfdl:
--output decoded:basestation:tcp:mode=server,address=192.168.10.140,port=29106

IP address should match to your running combine1090 system. It needs minimum of two position from an aircraft to be shown.

**Comment by madjuju** [comment]
Yes readsb

Merci l try
